manifests/w/wgyt/wgyttk/20211205-192649-672c1cc1/wgyt.wgyttk.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created with YamlCreate.ps1 v2.0.2 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: wgyt.wgyttk
+PackageVersion: 20211205-192649-672c1cc1
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: user
+InstallModes:
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/wez/wezterm/releases/download/20211205-192649-672c1cc1/WezTerm-20211205-192649-672c1cc1-setup.exe
```

Comment:
This seems like the wrong url, i can't find this package and it dosn't look like this one https://github.com/microsoft/winget-pkgs/tree/master/manifests/w/wez/wezterm